@xen-orchestra/lite/src/components/ui/UiCardTitle.vue

```diff
@@ -6,6 +6,7 @@
       class="left"
     >
       <slot>{{ left }}</slot>
+      <UiCounter class="count" v-if="count > 0" :value="count" color="info" />
```

Comment:
We can add something more generic like an info slot.

Reply:
An info slot would be too generic I think.

I'll add a comment on Figma to make sure we want this kind of inconsistency regarding counters in card titles.

For example:
- Tasks have a round counter next to the title
- Alarms have a round counter at the opposite of the title
- Patches have a "text" counter ("3 missing") at the opposite of the title

Reply:
I'm guessing we're keeping it like this for now, then?

Reply:
Clémence has to decide what to do about it. I think we can leave it at that for now and make changes later if necessary.